src/test/e2e/api/admin/project/projects.e2e.test.ts

```diff
@@ -36,3 +36,26 @@ test('Should ONLY return default project', async () => {
     expect(body.projects).toHaveLength(1);
     expect(body.projects[0].id).toBe('default');
 });
+
+test('Should store and retrieve default project stickiness', async () => {
+    const appWithDefaultStickiness = await setupAppWithCustomConfig(db.stores, {
+        experimental: {
+            flags: {
+                projectScopedStickiness: true,
```

Comment:
Please add 
```
experimental: 
{
            flags: {
                strictSchemaValidation: true,
            },
        },
```
for new code so that we can validate schema from the very beginning